cypress/support/step-definitions/decimal-steps.js

```diff
@@ -1,65 +1,70 @@
 import { inputPrecisionSlider } from '../../locators/decimal/index';
 import { setSlidebar } from '../helper';
-import { label, commonDataElementInputPreview } from '../../locators';
+import { label, commonDataElementInputPreviewNoIframe, getDataElementByValueNoIframe } from '../../locators';
 
 const TEXT_ALIGN = 'text-align';
 
 Then('input direction is {string}', (direction) => {
```

Comment:
IF it's only for `Decimal`, please rename this step e.g `Decimal input direction is {string}`. If it's a reusable step for other components, move it to `common` file

Reply:
moved to `common-steps.js` file